lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -28,6 +28,7 @@
 use Doctrine\Persistence\Mapping\ReflectionService;
 use InvalidArgumentException;
 use ReflectionClass;
+use ReflectionProperty;
```

Comment:
It's odd that we are mixing import and fcqn. Property uses \ReflectionProperty

Reply:
That shall be fixed in #8144